### ISSUE TITLE
Contrast errors present on all report headings, table headings and Heritage Asset title

### ISSUE THREAD

**Issue by csmith-he** [archesproject/Arches_HER#1028]
**Steps to Reproduce:**
Navigate to Resource Editor
Enable WAVE browser extension to view contrast errors

**Expected Result:**
No contrast errors should be present so that users can discern all page content. 

**Actual Result:**
23 contrast errors are present. These occur on:
Heritage Asset page title
All report tab headings
All table headings

**Evidence:**

![image](https://user-images.githubusercontent.com/38533555/191951424-583e3296-1cd6-48b0-951b-a20b3845d942.png)

**System Info**

Chrome Version 102.0.4951.67 (Official Build) (64-bit)
Firefox 91.9.0esr (64-bit)
Edge Version 101.0.1210.47 (Official build) (64-bit)
Safari 12.1
